node/sc/bridge_manager.go

```diff
@@ -12,7 +12,6 @@
 // GNU Lesser General Public License for more details.
 //
 // You should have received a copy of the GNU Lesser General Public License
-// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
What is the reason we remove this?

Reply:
I've no idea why it was removed because I didn't touch it if I correctly remember it. Recovered. Thank you.